Test/CoreSDK.Test/Shared/Extensibility/Implementation/TaskTimerInternalTest.cs

```diff
@@ -133,23 +133,30 @@ public void InvokesActionAfterDelay()
             }
 
             [TestMethod]
-            public void CancelsPreviousActionWhenStartIsCalledMultipleTimes()
+            public void StartIsCalledMultipleTimes()
```

Comment:
nitpick: Should be slightly more descriptive new name. e.g. EachCallStartsSeparateTimerTaskWhenStartIsCalledMultipleTimes.